### DISCUSSION TITLE
unable to import a pool

### DISCUSSION THREAD

**Discussion by evilgausus** [zbm-dev/zfsbootmenu#620] [Questions]
Hello :slight_smile:

I have run into some issues with zfsbootmenu after reboot. After the UEFI phase zfsbootmenu starts and displays an error “Unable to import pool, retryin in” - and keeps retrying.
I have checked using a rescue disk and the pool is fine.

zreport command show that there are no pools or datasets available.

What might be the issue here and how can i fix it?
Thank you in advance for any help :slight_smile:

**Comment by ahesford** [comment]
Can you import the pool manually from the ZFSBootMenu emergency shell?
